integration-testing/bsd/cypress/e2e/configuration.cy.ts

```diff
@@ -46,16 +46,19 @@ describe('BSD and services/Scan', () => {
 
   it('BSD can be configured with services/scan with an election JSON file', () => {
     cy.contains('Load Election Configuration');
-    cy.get('input[type="file"]').attachFile('election.json');
+    cy.get('input[type="file"]').selectFile('cypress/fixtures/election.json', {
+      force: true,
```

Comment:
These `force: true` options were required because we use the "hidden input with a button or label" technique, so cypress sees the input as covered or inaccessible and will error by default without this.